src/D2L.Bmx/AwsCredsCreator.cs

```diff
@@ -62,6 +64,12 @@ bool nonInteractive
 			}
 		}
 
-		return await awsClient.GetTokensAsync( samlResponse, selectedRoleData, duration.Value );
+		return await awsClient.GetTokensAsync(
+			samlResponse,
+			selectedRoleData,
+			duration.Value,
+			useCache,
+			config.Org ?? "",
+			config.User ?? "" );
```

Comment:
If you use string.IsNullOrEmpty in the above suggestion then you can just pass in config.Org config.User